.changes/unreleased/tinymce-TINY-11488-2024-11-11.yaml

```diff
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Added
+body: New `disabled` option to restore the previous `readonly` mode behavior, allowing
```

Comment:
`disabled` was one of those words that was considered offensive so everything was swapped to `enabled` for all apis etc. So I think we need to flip this is we really still care about that see TINY-8101 for details.

Reply:
Do we really need to make this one so complex couldn't this simply be a setting that controls what the readonly mode does. Like a `readonly_disabled` option that can't be mutated independently it just controls what the readonly mode toggle does if it disables everything or just somethings and leave caret movement etc?

Reply:
The requirement was made to make the `disabled` option like the previous readonly mode, which includes updating the option at runtime. I'm wondering if introducing a separate `disabled` option might be clearer here 🤔

Reply:
I don't think anyone is particularly attached to the `disabled` name. But that would be worth double-checking.

We can't really invert it to `editor.enabled = false` like we did with the other `disabled` APIs. I wonder if it could be something like `editor.lockdown` or `editor.blocked` 🤔

Reply:
Why can't we invert it if it all goes though a isDisabled function intent ally just flip the logic in that and call the option enabled and make it on by default?

Reply:
We could also expose the isEnabled on either mode or maybe even editor that way you can't mutate it since it's just a getter predicate.

Reply:
We are going with `disabled` option as proposed, see comments in EPIC-165

Reply:
well that's confusing, because after you wrote it the decision changed 😂